### DISCUSSION TITLE
Documentation not working

### DISCUSSION THREAD

**Discussion by salilponde** [l-lin/angular-datatables#1746] [General]
I am trying to view the documentation at https://l-lin.github.io/angular-datatables. After I open any page, the code is displayed for a fraction of a second and then it hides. So the documentation is not usable.

I tried on Chrome, Edge, Firefox. All same.

**Comment by shanmukhateja** [comment]
Hi, can you attach a screenshot of what you are referring to?

**Comment by salilponde** [comment]
There is content in the red area but it doesn't appear. It shows for a split of a second and then disappears.

![image](https://github.com/l-lin/angular-datatables/assets/831441/6e0cd268-a368-419a-9b3c-2b7f9f03a8de)

**Reply by shanmukhateja** [reply]
Can you confirm if the missing data isn't part of "HTML" or "TS" tabs? You can click on the tabs to view their contents.

**Reply by salilponde** [reply]
Oh, ok. Thanks.
